.github/workflows/rust.yml

```diff
@@ -0,0 +1,15 @@
+name: Rust
```

Comment:
Does simply naming the action `Rust` ensure that rust/cargo are installed?

Reply:
Rust is included in the base Ubuntu image in GitHub Actions.